packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -1108,3 +1191,7 @@ const getLegendProps = (data: Data[], layout: Partial<Layout> | undefined) => {
     hideLegend: layout?.showlegend === false ? true : hideLegends,
   };
 };
+
+const isInvalid = (value: any) => {
```

Comment:
isInvalidNumber

Reply:
renamed to isInvalidValue as it can be used to check for undefined or null values in string data as well